docs/guide/troubleshooting.rst

```diff
@@ -56,4 +56,5 @@ Save this configuration file as ``debug_logging.conf`` and then run ``openfe qui
    The ``--log debug_logging.conf`` argument goes between ``openfe`` and ``quickrun``.
 
 This will cause every package to log at the debug level, which may be quite verbose and noisy but should aid in identify what is going on right before the exception is thrown.
-We do not recommend setting the log level to debug for production runs as the logging may slow down the simulation and add a lot of noise to the output.
+
+**We do not recommend setting the log level to debug for production runs,** as the logging may slow down the simulation and add a lot of noise to the output.
```

Comment:
Wondering if this should just be bold, or if we want to use a .. warning:: style callout box?

Reply:
added it as a warning and moved to toward the top, but there's two other notes already and so it's kind of a "if everything is emphasized, nothing is emphasized" kind of problem.